manifests/o/OpenMusic/OpenMusic/7.0/OpenMusic.OpenMusic.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: OpenMusic.OpenMusic
+PackageVersion: 7.0
+PackageLocale: en-US
+Publisher: Ircam

```

```suggestion
Publisher: IRCAM
```
This must match Control Panel.